src/js/components/stock-movement-wizard/request/AddItemsPage.jsx

```diff
@@ -821,7 +828,7 @@ class AddItemsPage extends Component {
         val => ({
           ...val,
           disabled: true,
-          quantityOnHand: this.state.isRequestFromWard ? '' : val.quantityOnHand,
+          quantityOnHand: val.quantityCounted,
```

Comment:
I'm not sure if it is correct? 
Do you have confirmed behaviors for:

- locations which are not ward 
- stocklist both types: pull, push?
- stock request without using a stocklist
?

Reply:
I forgot about locations which are not ward. Thanks for catching that.